layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -479,15 +479,61 @@ If the universal prefix argument is used then will the windows too."
             (if dedicated "no longer " "")
             (buffer-name))))
 
-;; http://camdez.com/blog/2013/11/14/emacs-show-buffer-file-name/
-(defun spacemacs/show-and-copy-buffer-filename ()
-  "Show and copy the full path to the current file in the minibuffer."
+
+;; Copy file path
+(defun spacemacs/copy-directory-path ()
+  "Copy and show the directory path of the current buffer.
+If the buffer is not visiting a file, use the `list-buffers-directory'
+variable as a fallback to display the directory, useful in buffers like the
+ones created by `magit' and `dired'."
+  (interactive)
+  (if-let (directory-name (or
+                            (if-let (file-name (buffer-file-name))
+                              (file-name-directory file-name))
+                            list-buffers-directory))
+    (message "%s" (kill-new (file-truename directory-name)))
```

Comment:
Usually the then form is indented two spaces more than the else form.  Might be worth doing an auto-indent on this file.

Reply:
I'm using `evil-lisp-state` from time to time and it reindents everything like this. Maybe there's a setting that controls this behavior, I'll try to see if I can get it to indent properly by default.